casper/src/test/scala/coop/rchain/casper/addblock/MultiParentCasperAddBlockSpec.scala

```diff
@@ -39,39 +38,39 @@ class MultiParentCasperAddBlockSpec extends FlatSpec with Matchers with Inspecto
 
   //put a new casper instance at the start of each
   //test since we cannot reset it
-  "MultiParentCasper" should "not allow multiple threads to process the same block" in {
-    val scheduler = Scheduler.fixedPool("three-threads", 3)
-    val testProgram =
-      TestNode.standaloneEff(genesis)(scheduler).use { node =>
-        val casper = node.casperEff
-        for {
-          deploy <- ConstructDeploy.basicDeployData[Effect](0)
-          _      <- casper.deploy(deploy)
-          block <- casper.createBlock.map {
-                    case Created(block) => block
-                    case _              => throw new RuntimeException()
-                  }
-          result <- Task
-                     .racePair(
-                       casper.addBlock(block),
-                       casper.addBlock(block)
-                     )
-                     .flatMap {
-                       case Left((statusA, running)) =>
-                         running.join.map((statusA, _))
-
-                       case Right((running, statusB)) =>
-                         running.join.map((_, statusB))
-                     }
-        } yield result
-      }
-    val threadStatuses: (ValidBlockProcessing, ValidBlockProcessing) =
-      testProgram.unsafeRunSync(scheduler)
-
-    threadStatuses should matchPattern {
-      case (Left(CasperIsBusy), Right(Valid)) | (Right(Valid), Left(CasperIsBusy)) =>
-    }
-  }
+//  "MultiParentCasper" should "not allow multiple threads to process the same block" in {
+//    val scheduler = Scheduler.fixedPool("three-threads", 3)
+//    val testProgram =
+//      TestNode.standaloneEff(genesis)(scheduler).use { node =>
+//        val casper = node.casperEff
+//        for {
+//          deploy <- ConstructDeploy.basicDeployData[Effect](0)
+//          _      <- casper.deploy(deploy)
+//          block <- casper.createBlock.map {
+//                    case Created(block) => block
+//                    case _              => throw new RuntimeException()
+//                  }
+//          result <- Task
+//                     .racePair(
+//                       casper.addBlock(block),
+//                       casper.addBlock(block)
+//                     )
+//                     .flatMap {
+//                       case Left((statusA, running)) =>
+//                         running.join.map((statusA, _))
+//
+//                       case Right((running, statusB)) =>
+//                         running.join.map((_, statusB))
+//                     }
+//        } yield result
+//      }
+//    val threadStatuses: (ValidBlockProcessing, ValidBlockProcessing) =
+//      testProgram.unsafeRunSync(scheduler)
+//
+//    threadStatuses should matchPattern {
+//      case (Left(CasperIsBusy), Right(Valid)) | (Right(Valid), Left(CasperIsBusy)) =>
+//    }
+//  }
```

Comment:
Is the refactoring of this part mentioned in some issue?

Reply:
https://github.com/rchain/rchain/issues/3277#issuecomment-798109618